Added base styles

sanketbansal opened the pull request:
<!--
(Thanks for sending a pull request! .)
-->

<!--The PR title should start with "Fix #bugnum: " (if applicable), followed by a clear one-line present-tense summary of the changes introduced in the PR. For example: "Fix #bugnum: Introduce the first version of the collection editor." -->

<!-- Add issue number here. If you do not solve the issue entirely, please change the message e.g. "Addresses #IssueNumber -->

#### Changes proposed in this pull request:

<!-- Changes: Add here what changes were made in this issue and if possible provide links. -->
- Added base styles required for different components in the project.

<!-- Demo Link: Add here the link where you changes can be seen. -->
- Link to live demo: http://pr-182-evalai.surge.sh <!-- Replace XXX with your PR no: -->

<!-- Screenshots for the change: Add here the screenshot of the fix. -->

sanketbansal committed:
Added basic styles

Basic style files added in styles-port folder

sanketbansal committed:
Removed duplicate styles

Removed duplicate scss files from basic.scss

sanketbansal committed:
Changes in angular.json

Added basic.scss styles globally

sanketbansal committed:
Removed styles from styles-port folder

basic.scss has been removed and added into the base.scss file

lunayach committed:
Merge branch 'master' into base-styles

lunayach commented on src/styles/base.scss:
```diff
@@ -1555,3 +1555,592 @@ td, th {
+  }
+}
+
+.list-icon {
```
No need of this, I guess.

sanketbansal replied:
okay, I will check that

Sanji515 commented on src/styles/base.scss:
```diff
@@ -1555,3 +1555,592 @@ td, th {
+  -webkit-text-fill-color: #4d4d4d !important;
+}
+
+input[type="radio"].selectTeam + label {
```
@sanketbansal `radio button` styles are already present in the `base.scss` here https://github.com/Cloud-CV/EvalAI-ngx/blob/master/src/styles/base.scss#L694. Can you please take a look and modify accordingly so that the `submit page` UI do not gets break.

sanketbansal replied:
@Sanji515  I do not see any break in the UI of submit page.
is it breaking on your side?

Sanji515 replied:
At least you should keep these at one place

sanketbansal replied:
okay

Sanji515 commented on src/styles/base.scss:
```diff
@@ -1555,3 +1555,592 @@ td, th {
+  background-color: rgba(0, 0, 0, 0.1);
+}
+
+.ev-btn-dark:focus {
```
how about if you keep this css with other `ev-btn-dark` as it's already here https://github.com/Cloud-CV/EvalAI-ngx/blob/master/src/styles/base.scss#L542

Sanji515 commented on src/styles/base.scss:
```diff
@@ -1555,3 +1555,592 @@ td, th {
+
+.w-400 {
+  font-weight: 400;
+}
```
@sanketbansal No need to add these fonts weight as they are already present here https://github.com/Cloud-CV/EvalAI-ngx/blob/master/src/styles/base.scss#L127. You can use these

Sanji515 replied:
@sanketbansal No need to add these fonts weight as they are already present here https://github.com/Cloud-CV/EvalAI-ngx/blob/master/src/styles/base.scss#L127. You can use these

Sanji515 commented on src/styles/base.scss:
```diff
@@ -1555,3 +1555,592 @@ td, th {
+  padding-bottom: 20px;
+}
+
+.btn {
```
@sanketbansal this will conflict with this here https://github.com/Cloud-CV/EvalAI-ngx/blob/master/src/styles/base.scss#L1157

sanketbansal replied:
It won't I have checked that.

Sanji515 commented on src/styles/base.scss:
```diff
@@ -1555,3 +1555,592 @@ td, th {
+
+.input-field input[type=email]:focus+.form-icon-dark,
+.input-field input[type=password]:focus+.form-icon-dark,
+.input-field input[type=text]:focus+.form-icon-dark {
```
There's no need to add these input styles here, as I've already added in the `input` component scss file, @sanketbansal Please take a look at those

sanketbansal replied:
I checked my code and I am using these styles as I have some customized input elements.

Sanji515 replied:
I think at all the places we are having the same input containing same css and even if you want to customize it then it would be better if you could do it in the existing one i.e. inside input component scss file.

sanketbansal replied:
Let's keep it like that now. we can modify into that later.

sanketbansal committed:
Removed commented styles

styles which are commented are removed.

sanketbansal committed:
Changes in base styles.scss

Placed .ev-btn-dark styles together
Removed .btn style to resolve possible conflict
Removed w-400, w-300 styles
Moved text color style at one place
Moved input type="radio" styles together with other label styles.
Removed duplicate input type=checkbox property

Sanji515 commented on src/styles/base.scss:
```diff
@@ -124,11 +124,11 @@ body {
 }
 
-.fw-regular {
+.fw-regular, .w-400 {
```
@sanketbansal Why are you adding this as duplicate names for using this font?, instead use `fw-light` or `fw-regular` in the places where you have used it

sanketbansal replied:
Because now I have to search these styles in each and every PR I have made and made the insignificant change.

Sanji515 committed:
Merge branch 'master' into base-styles

Sanji515 commented:
@RishabhJain2018 @lunayach LGTM! :tada: Now it can be merged.

lunayach commented:
Thanks, @Sanji515 for verifying the changes!
